express-get-json/index.js

```diff
@@ -0,0 +1,30 @@
+var express = require('express');
+var app = express();
+var grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+var temp = [];
+app.listen(3000);
+app.get('/api/grades', (req, res) => {
+  for (var k in grades) {
+    temp.push(grades[k]);
+  }
```

Comment:
Try moving your array into this callback function. Otherwise every request makes the array bigger.